src/webgui/src/webgui/image_converter.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```

Comment:
We probably don't need this anymore, since ros has a node that does the same thing anyways.

Reply:
Should I just delete the file?

Reply:
Yeah, unless you can think of a good reason to keep it